src/features/user/Account/components/AccountRecord.tsx

```diff
@@ -36,11 +36,22 @@ export default function AccountRecord({
   isModal = false,
 }: Props): ReactElement {
   const { t } = useTranslation(['me']);
+  const [showCertificate, setShowCertificate] = React.useState<boolean>(false);
 
   const outerDivClasses = isModal
     ? styles.recordModal
     : `${styles.record} ${selectedRecord === index ? styles.selected : ''}`;
 
+  React.useEffect(() => {
+    if (
+      record?.details?.donorCertificate ||
+      record?.details?.taxDeductibleReceipt ||
+      record?.details?.giftCertificate
+    ) {
+      setShowCertificate(true);
+    }
+  }, [record]);
+
```

Comment:
@sunilsabatp 
1. Do you need a `useEffect` here? The value of `showCertificate` won't change for a `record` once it is passed as a prop, and is already available to the component.
2. You could even avoid using state and just use a regular variable for `showCertificate` as the value won't change. If you're using state to prevent recalculation between renders, perhaps useMemo is a better option. 
https://reactjs.org/docs/hooks-reference.html#usememo